DataManagementSystem/Utilities/DMSHelpers.py

```diff
@@ -102,10 +116,10 @@ def getSiteSEMapping( self ):
       for site in sites:
         candidateSEs = gConfig.getValue( '/Resources/Sites/%s/%s/SE' % ( grid, site ), [] )
         if candidateSEs:
+          for se in list( candidateSEs ):
```

Comment:
Maybe something like : 
candidateSEs.extend([ eqSe for se in candidateSEs for eqSe in equivalentSEs.get(se,[])])
is less tricky, because one day, someone "smart" could think that list(canidateSEs) is pointless because it is already a list. 
OR a comment ;)